src/commands/list_folder.ts

```diff
@@ -36,14 +32,14 @@ new CLICommand({
 			const arDrive = new ArDrive(
 				wallet,
 				cliWalletDao,
-				new ArFSDAO(wallet, arweave),
-				new ArDriveCommunityOracle(arweave),
+				new ArFSDAO(wallet, cliArweave),
+				new ArDriveCommunityOracle(cliArweave),
```

Comment:
Shouldn't we be getting these from injection? Maybe move it to the context class???

Reply:
For this question, I think you are the guy @arielmelendez! ;)

Reply:
Outdated. Closing.